### DISCUSSION TITLE
Run 2 Tauons flatpak at the same time?

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#811] [Q&A]
Is it possible to run 2 Tauons flatpak (stable and beta) at the same time? I tried running them from the terminal, but only one instance is allowed it seems. 

I guess you can manually create a tauon-beta shortcut in the menu with menulibre app, but it won't have a Tauon icon.

**Comment by Taiko2k** [comment]
You would launch specific branches using `flatpak run --branch=beta com.github.taiko2k.tauonmb` where *beta* can also be *stable*. But these's currently no flag to disable Tauons single instance enforcement.

**Reply by Solarunit** [reply]
gotcha, thanks

**Comment by RSKYS** [comment]
Hey there,
I think this thread belongs to long time ago but, I should mention that it's possible in nix anyway

(I was just passing by, don't open up this part much)
